tasklist/client/e2e/pages/ProcessesPage.ts

```diff
@@ -17,12 +17,18 @@ class ProcessesPage {
   readonly searchProcessesInput: Locator;
   readonly processTile: Locator;
   readonly tasksTab: Locator;
+  readonly startProcessSubButton: Locator;
 
   constructor(page: Page) {
     this.page = page;
     this.continueButton = page.getByRole('button', {name: 'Continue'});
     this.cancelButton = page.getByRole('button', {name: 'Cancel'});
     this.startProcessButton = page.getByRole('button', {name: 'Start process'});
+    this.startProcessSubButton = page
+      .getByRole('button', {
+        name: 'Start process',
+      })
+      .last();
```

Comment:
This the complete button inside the modal? If yes let's remove the `.last()` since it's not reliable, and just query for the button inside the modal

Let's also rename it to `modaLStartProcessButton`

```suggestion
```